kpops/cli/config.py

```diff
@@ -45,7 +81,7 @@ class PipelineConfig(BaseSettings):
         description="The environment you want to generate and deploy the pipeline to. "
         "Suffix your environment files with this value (e.g. defaults_development.yaml for environment=development).",
     )
-    brokers: str = Field(
+    kafka_brokers: str = Field(
```

Comment:
isn't this also a comma separated string of URLs? would it make sense to deserialize it to a `list[HttpUrl]` for example?

Reply:
We cannot treat broker addresses as HttpUrl. The don't have necessary the schema of an URL. They look like this: `bla0-buba-ee.dubadbabadie.zy.internal.myapp.net:9092`